src/parsers/gas.js

```diff
@@ -119,19 +121,40 @@ export const defaultGasPriceFormat = (option, timeWait, value) => {
  * @param {Object} prices
  * @param {Number} gasLimit
  */
-export const parseTxFees = (gasPrices, priceUnit, gasLimit, nativeCurrency) => {
+export const parseTxFees = (
+  gasPrices,
+  priceUnit,
+  gasLimit,
+  nativeCurrency,
+  l1GasFeeOptimism
+) => {
   const txFees = map(GasSpeedOrder, speed => {
     const gasPrice = get(gasPrices, `${speed}.value.amount`);
-    const txFee = getTxFee(gasPrice, gasLimit, priceUnit, nativeCurrency);
+    const txFee = getTxFee(
+      gasPrice,
+      gasLimit,
+      priceUnit,
+      nativeCurrency,
+      l1GasFeeOptimism
+    );
     return {
       txFee,
     };
   });
   return zipObject(GasSpeedOrder, txFees);
 };
 
-const getTxFee = (gasPrice, gasLimit, priceUnit, nativeCurrency) => {
-  const amount = multiply(gasPrice, gasLimit);
+const getTxFee = (
+  gasPrice,
+  gasLimit,
+  priceUnit,
+  nativeCurrency,
+  l1GasFeeOptimism
+) => {
+  let amount = multiply(gasPrice, gasLimit);
+  if (l1GasFeeOptimism && greaterThan(l1GasFeeOptimism, '0')) {
```

Comment:
you need `greaterThan` from `react-native-reanimated` ? the one in `utilities` isn't good enough for u?